### PR TITLE
Clarify double cell type and performance note

### PR DESCRIPTION
**Pull request by kkraune**
I assume _tensorFromLabels_ also uses double cell type

### CODE REVIEW

**Review by baldersheim — APPROVED** [review]
Perfect, @kkraune you eyes in the neck.

**Comment by baldersheim** [comment]
@havardpe is @kkraune correct about tensorFromLabels ?

File: en/reference/rank-features.html [hunk 1]
```diff
@@ -164,40 +164,47 @@ <h2 id="feature-list">Feature list</h2>
...
 </pre>
-    <em>tensorFromWeightedSet(attribute(myField), dim)</em> produces:
+    <p><em>tensorFromWeightedSet(attribute(myField), dim)</em> produces:</p>
 <pre>
-tensor(dim{}):{ {dim:key1}:0.0, {dim:key2}:1.0, {dim:key3}:2.5} }
+tensor&lt;double&gt;(dim{}):{ {dim:key1}:0.0, {dim:key2}:1.0, {dim:key3}:2.5} }
 </pre>
+    {% include note.html content="This creates a temporary tensor on the fly,
+    and has build cost and extra memory touched.
+    If using other cell types, there is also conversion cost.
```
[the remark this answers is not in the document]
Comment (kkraune) [reply en/reference/rank-features.html]: good points! PTAL

[the remark this answers is not in the document]
Comment (kkraune) [reply en/reference/rank-features.html]: ... I'll just merge, thanks for feedback @havardpe

**Pull request merged**